### PR TITLE
MIG-813: MTC 1.6.0 release notes

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIG-813
https://bugzilla.redhat.com/show_bug.cgi?id=1963927

Release notes

4.6+

Preview: https://deploy-preview-36096--osdocs.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/mtc-release-notes.html#migration-mtc-release-notes-1-6_mtc-release-notes

QE approved. Ready for peer review.

### CODE REVIEW

**Comment by apinnick** [comment]
@XinRedhat  Please review.

File: modules/migration-mtc-release-notes-1-6.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+The release notes for {mtc-full} ({mtc-short}) version 1.6 describe new features and enhancements, deprecated features, and known issues.
+
+[id="new-features-and-enhancements-1-6_{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+* State migration
++
+You can migrate an application's state by selecting persistent volume claims (PVCs) and Kubernetes objects for migration.
```
Comment (eriknelson) [inline modules/migration-mtc-release-notes-1-6.adoc:17]: Normal migrations have always migrated state. The way this is worded suggests this is a new feature to me. I think I would stress, with this release we've added new functionality that allows for ad-hoc, repeatable state only transfers on a PVC by PVC basis.

File: modules/migration-mtc-release-notes-1-6.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* {mtc-short} version 1.4 is no longer supported.
+
+[id="known-issues-1-6_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+* On {product-title} 3.10, the `MigrationController` pod takes too long to restart. The bug report contains a workaround. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986796[*BZ#1986796*])
+
+* Direct volume migration from a classic OpenShift cluster on IBM Cloud fails unless the application is stopped during migration. The IBM block storage plugin does not allow the same volume to be mounted on multiple pods of the same node. As a result, PVCs cannot be mounted on the Rsync pods and on the application pods simultaneously.
```
Comment (eriknelson) [inline modules/migration-mtc-release-notes-1-6.adoc:37]: To unblock this, the workload can be quiesced prior to a migration to ensure that there is only ever one pod mounting an RWO volume.

**Review by eriknelson — CHANGES_REQUESTED** [review]
I'm seeing a blank section for "Technical Changes", what normally goes in this section?

**Comment by sagidlow** [comment]
LGTM!

**Review by eriknelson — APPROVED** [review]
LGTM

**Comment by apinnick** [comment]
@XinRedhat Can you review the last item in "known issues"? BZ1963927

**Comment by XinRedhat** [comment]
LGTM

**Review by skrthomas — APPROVED** [review]
LGTM!

**Comment by apinnick** [comment]
Releasing with module commented out so that I can create separate PRs for additions to the RN.

**Pull request merged**